### PR TITLE
Add version and fix README/CONTRIBUTE

### PR DESCRIPTION
**Pull request by itdove**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [comment]: # ( Copyright Contributors to the Open Cluster Management project )
 # clusteradm CLI & CLI Plugin
 
-A CLI and kubernetes CLI plugin that allows you to interact with OCM/ACM to provision and managed your Hybrid Cloud presence from the command-line.
+A CLI and kubernetes CLI plugin that allows you to interact with open-cluster-management to provision and managed your Hybrid Cloud presence from the command-line.
```
Comment (qiujian16) [inline README.md:4]: s/provision and managed/manage/

File: pkg/cmd/version/exec.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	fmt.Printf("client\t\tversion\t:%s\n", clusteradm.GetVersion())
+	client, err := helpers.GetControllerRuntimeClientFromFlags(o.ConfigFlags)
+	if err != nil {
+		return err
+	}
+	return o.runWithClient(client)
+}
+
+func (o *Options) runWithClient(client crclient.Client) (err error) {
+	cms := &corev1.ConfigMapList{}
```
Comment (qiujian16) [inline pkg/cmd/version/exec.go:37]: let's empty  this function. I think it tries to get image-manfest in ACM which will not work.

File: sonar-project.properties [hunk 3]
```diff
@@ -1,5 +1,5 @@
-sonar.projectKey=open-cluster-management_cm-cli
-sonar.projectName=cm-cli
+sonar.projectKey=open-cluster-management_clusteradm
```
Comment (qiujian16) [inline sonar-project.properties:1]: let's remove sonar file until we decide whether we also need to integrate sonar here.

File: README.md (hunk 1 above)
Comment (itdove) [reply README.md:4]: done

File: pkg/cmd/version/exec.go (hunk 2 above)
Comment (itdove) [reply pkg/cmd/version/exec.go:37]: it works, if image-manifest not present the list is empty and so jump here: https://github.com/itdove/clusteradm/blob/bd9c173b4231918e5a3928d85e590c584cb8ce6f/pkg/cmd/version/exec.go#L68

on kind:

(base) dvernier@dvernier-mac ~ % clusteradm version
client		version	:0.0.1
server release	version	:v1.19.7

As per the arch call this morning, I understood that we have to do function that works for both products.

File: sonar-project.properties (hunk 3 above)
Comment (itdove) [reply sonar-project.properties:1]: Done

File: pkg/cmd/version/exec.go (hunk 2 above)
Comment (itdove) [reply pkg/cmd/version/exec.go:37]: There is also a e2e test on it which run on kind: https://github.com/itdove/clusteradm/blob/6ca5416204fd1b207082bc103695a03232adbe32/build/run-functional-tests.sh#L20

Comment (qiujian16) [reply pkg/cmd/version/exec.go:37]: i think we should remove wording like `acm-custom-registry` from the code. 
If the intention is to fetch the server version. We need  a  better approach instead  of reading it from image manifest...So it is better to make this an empty function at first.

Comment (itdove) [reply pkg/cmd/version/exec.go:37]: I will just remove the ACM part of it

**Comment by qiujian16** [comment]
thanks please squash the commit, then we can merge this.

**Comment by itdove** [comment]
Can't squash and merge in GitHub directly ?

**Comment by qiujian16** [comment]
I haven't enable squash in openshift-merge-bot yet....And I would want openshift-merge-bot to merge PRs....

**Comment by itdove** [comment]
Close as to complex to squash, will open a new one.

**Pull request closed without merging**